### PR TITLE
resizable pannels, map/search result placeholders

### PR DESCRIPTION
**Pull request by JacobZwang**
Built basic UI for the home page. The panels are resizable and the search results (sort of) resize with them. The next steps are to refine ui, add more data displays, hook up deckgl, and use fake api rather than hardcoded values. 
Screenshot 2020-09-11 at 7 48 24 PM

### CODE REVIEW

File: client/web/public/index.html [hunk 1]
```diff
@@ -2,6 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
+    <link rel="stylesheet" href="main.css" />
```
Comment (theswerd) [inline client/web/public/index.html:5]: please bring the CSS into the React JSX

File: client/web/public/main.css [hunk 2]
```diff
@@ -0,0 +1,16 @@
+* {
```
Comment (theswerd) [inline client/web/public/main.css:1]: Please move this to the src and add it to the `index.js` file

File: client/web/src/components/SearchResult.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+export default function SearchResult() {
+  return (
+    <div style={{ backgroundColor: "#ffffff", margin: "5px", padding: "10px" }}>
```
Comment (theswerd) [inline client/web/src/components/SearchResult.js:5]: Please either extract these into a CSS file

File: client/web/src/pages/Home.js [hunk 4]
```diff
@@ -1,10 +1,60 @@
...
+import Background from "../assets/mapPlaceholder.png";
+import Logo from "../assets/ballotnav-logo.svg";
+import SearchResult from "../components/SearchResult";
 
 function Home() {
   const { id } = useParams();
   const hasID = id !== undefined;
-  return <div>home {hasID? "id: " + id:"" }</div>;
+  return (
+    <div style={{ height: "100vh", overflow: "hidden" }}>
```
Comment (theswerd) [inline client/web/src/pages/Home.js:13]: Bring these into a CSS file

**Review by theswerd — CHANGES_REQUESTED** [review]
See notes re-CSS

File: client/web/public/index.html (hunk 1 above)
Comment (JacobZwang) [reply client/web/public/index.html:5]: I don't know how css works with react, can you handle that?

File: client/web/public/index.html [hunk 5]
```diff
@@ -26,6 +26,7 @@
     -->
     <title>React App</title>
   </head>
+
```
Comment (theswerd) [inline client/web/public/index.html:29]: remove extra space
```suggestion
```

**Review by theswerd — APPROVED** [review]

**Pull request merged**